subsys/bluetooth/services/hrs.c

```diff
@@ -79,6 +79,29 @@ static ssize_t read_blsc(struct bt_conn *conn, const struct bt_gatt_attr *attr,
 				 sizeof(hrs_blsc));
 }
 
+static ssize_t ctrl_point_write(struct bt_conn *conn, const struct bt_gatt_attr *attr,
+				const void *buf, uint16_t len, uint16_t offset, uint8_t flags)
+{
+	int err = -EAGAIN;
+	struct bt_hrs_cb *listener;
+
+	LOG_INF("HRS CTRL Point Written %d", len);
+
+	SYS_SLIST_FOR_EACH_CONTAINER(&hrs_cbs, listener, _node) {
+		if (listener->ctrl_point_write) {
+			err = listener->ctrl_point_write(*((uint8_t *)buf));
+			if (err) {
+				break;
+			}
+		}
+	}
```

Comment:
This seems a bit tricky. 

So there are multiple listeners and it's just whomever returns a non-0 value first that gets to control the flow?